packages/application-components/src/components/public-page-layout/public-page-layout.tsx

```diff
@@ -41,6 +44,666 @@ const Container = styled.div`
   background-size: cover;
   background-position: center;
 `;
+const NewContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+
+  --1x: 25%;
+  --1y: 25%;
+  --2x: 75%;
+  --2y: 25%;
+  --3x: 25%;
+  --3y: 75%;
+  --4x: 75%;
+  --4y: 75%;
+  background: radial-gradient(
+      70% 70% at var(--3x) var(--3y),
+      var(--color-primary-90) 1%,
+      transparent 65%
+    ),
+    radial-gradient(
+      70% 70% at var(--2x) var(--2y),
+      var(--color-primary-90) 1%,
+      transparent 75%
+    ),
+    radial-gradient(
+      30% 90% at var(--4x) var(--4y),
+      var(--color-turquoise-90) 0% 5%,
+      transparent 80%
+    ),
+    radial-gradient(
+      30% 90% at var(--1x) var(--1y),
+      var(--color-turquoise-90) 0% 5%,
+      transparent 80%
+    );
+  background-blend-mode: color;
+  animation: gradients 10s ease both infinite;
+}
+
+@keyframes gradients {
```

Comment:
ok, looked at the figma, we do need the animations - maybe it would make sense to put this in its own file?

Reply:
We discussed - based on app kit practices it seems all other styled components are all directly in the file so we'll just keep everything in the file here

Reply:
+1 on keeping the `GradientBackgroundContainer ` in a separate file to improve readability.